src/dotnet/Fable.Core/Import/Node/Querystring.fs

```diff
@@ -0,0 +1,20 @@
+[<AutoOpen>]
+module rec Fable.Import.Node.Querystring
+
+open Fable.Core
+
+type [<AllowNullLiteral>] StringifyOptions =
+    abstract encodeURIComponent: (string -> string) option with get, set
+
+type [<AllowNullLiteral>] ParseOptions =
+    abstract maxKeys: float option with get, set
+    abstract decodeURIComponent: (string -> string) option with get, set
+
+type Globals =
+    abstract stringify: obj: 'T * ?sep: string * ?eq: string * ?options: StringifyOptions -> string
+    abstract parse: str: string * ?sep: string * ?eq: string * ?options: ParseOptions -> 'T
+    abstract escape: str: string -> string
+    abstract unescape: str: string -> string
+
+[<Import("*", "querystring")>] 
+let querystring: Globals = jsNative
```

Comment:
As discussed previously, we can rename this to `Exports` (maybe also the type becomes `IExports` instead of `Globals`) then users would do this (if the AutoOpen attribute is removed):

```fsharp
open Fable.Import.Node

QueryString.Exports.escape("my string")
```

We can also try to avoid `Exports` (or `Globals`) as in the old bindings. A solution without having to put everything in a single file, would be to add an extra file at the end with values like:

```fsharp
[<AutoOpen>]
module Fable.Import.Node.Exports

open Fable.Core

let QueryString: QueryString.Globals = jsNative
let Url: Url.Globals = jsNative
```

Then, after opening `Fable.Node.Import`, when users type `QueryString` they'll have both access to the types and the functions. What do you think?